prow/hook/server.go

```diff
@@ -43,6 +44,8 @@ type Server struct {
 	// c is an http client used for dispatching events
 	// to external plugin services.
 	c http.Client
+	// Used to tell hook when to stop gracefully
```

Comment:
nit: something like `// Tracks running handlers for graceful shutdown` is marginally better. "when to stop gracefully" is really the signal handler

Reply:
In 72cddfc